api/src/integration/java/org/pmiops/workbench/IntegrationTestConfig.java

```diff
@@ -0,0 +1,130 @@
+package org.pmiops.workbench;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.apache.ApacheHttpTransport;
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+import org.pmiops.workbench.auth.Constants;
+import org.pmiops.workbench.auth.ServiceAccounts;
+import org.pmiops.workbench.config.CommonConfig;
+import org.pmiops.workbench.config.RetryConfig;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.firecloud.ApiClient;
+import org.pmiops.workbench.firecloud.FireCloudConfig;
+import org.pmiops.workbench.google.CloudStorageService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.retry.backoff.BackOffPolicy;
+import org.springframework.retry.backoff.ExponentialRandomBackOffPolicy;
+import org.springframework.retry.backoff.Sleeper;
+import org.springframework.retry.backoff.ThreadWaitSleeper;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+import static org.springframework.context.annotation.FilterType.ASSIGNABLE_TYPE;
+
+@Configuration
```

Comment:
This config ends up repeating a lot of what's in the CommandLineTool config. It may behoove us to merge them at some point. I'll leave that for follow-up, probably combined w/ the work of pulling out our separate tools-specific Gradle config.